### PR TITLE
build : specify the output dir for ANTLR

### PR DESCRIPTION
**Pull request by ap891843**
specify the output dir for ANTLR

Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -349,7 +349,7 @@
                                 <argument>-package</argument>
                                 <argument>com.broadcom.usecase</argument>
                                 <argument>-o</argument>
-                                <argument>target/generated-sources/antlr4/com/broadcom/usecase</argument>
+                                <argument>${basedir}/target/generated-sources/antlr4/com/broadcom/usecase</argument>
```
Comment (grianbrcom) [inline server/pom.xml:352]: Yes, or even:
```
${project.build.directory}/generated-sources/antlr4/com/broadcom/usecase
```

**Review by grianbrcom — APPROVED** [review]
:dolphin:

File: server/pom.xml (hunk 1 above)
Comment (temanbrcom) [reply server/pom.xml:352]: ${basedir} was there before, I just didn't know why it should be there. I thought it is some obsolete value

**Review by temanbrcom — APPROVED** [review]
🏗️

File: server/pom.xml (hunk 1 above)
Comment (filipkroupa) [reply server/pom.xml:352]: it has to be there for build

**Pull request merged**